src/stores/GasPriceStore.js

```diff
@@ -73,15 +73,7 @@ class GasPriceStore {
         }
         return Promise.resolve()
       })
-      .catch(error => {
-        this.setProperty('slow', weiToGwei(GAS_PRICE.SLOW.PRICE))
-        this.setProperty('fast', weiToGwei(GAS_PRICE.FAST.PRICE))
-        this.setProperty('standard', weiToGwei(GAS_PRICE.NORMAL.PRICE))
-        this.setProperty('instant', weiToGwei(GAS_PRICE.INSTANT.PRICE))
-        this.setProperty('custom', weiToGwei(GAS_PRICE.CUSTOM.PRICE))
-
-        return Promise.reject(error)
-      })
+      .catch(error => Promise.reject(error))
```

Comment:
This doesn't do anything, does it? It's like doing `.then(x => x)`, if I'm not missing anything.

Reply:
fixed, thanks!